StockCount/Info.plist

```diff
@@ -20,20 +20,6 @@
 	<string>1.1</string>
 	<key>CFBundleVersion</key>
 	<string>1.1.1</string>
-	<key>Fabric</key>
```

Comment:
humm... not sure why this section has been removed. I think that's what we use when we deploy the app internally.

Reply:
- above fabric key value is used for Fabric integration only.
And fabric is used for only crash and log on debug version on my side.

Reply:
yes and we do use it for internal deployment. Our build script takes care to use the right key and secret.

Reply:
- buildscript for fabric will run only when I add fabric to pod and try to build when I debug.
There's no problem for release version, I think.

Reply:
Yes so we need Fabric back, we need to keep the way the info.plist was because our build process will take care of doing the right thing to deploy on Crashlytics. And it should not impact your build.  One that topic, your credentials in the project signing setting should not override ours. So when you commit the project file you should cherry pick the changes to make sure our signing/build will still work.

Reply:
Yes so we need Fabric back, we need to keep the way the info.plist was because our build process will take care of doing the right thing to deploy on Crashlytics. And it should not impact your build.  One that topic, your credentials in the project signing setting should not override ours. So when you commit the project file you should cherry pick the changes to make sure our signing/build will still work.

Reply:
revered fabric code remove.
Please review and let me know if there's any other issue more.